tests/integration/test_packages.py

```diff
@@ -28,6 +28,8 @@
         ("gomod_packages", "vendored_with_flag"),
         ("gomod_packages", "implicit_gomod"),
         ("gomod_packages", "missing_gomod"),
+        ("gomod_packages", "force_tidy_without_deps"),
+        ("gomod_packages", "force_tidy_vendored"),
```

Comment:
@MartinBasti 
>But why? It fixes tests so after checkout to this particular commit, tests won't be broken. Tests should be green on every commit, it shouldn't work in a way that test are fixed in followup commit

Integration tests will not be broken because there were no tests for this flag yet. It's fine not to separate work but I hoped it will be easier to read.